### PR TITLE
Adding McFly

### PR DESCRIPTION
**Pull request by kenwheeler**

### CODE REVIEW

**Comment by voronianski** [comment]
@kenwheeler merged!

File: mcfly/js/stores/CartStore.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+}, function(payload) {
+
+    switch (payload.actionType) {
+        case 'ADD_TO_CART':
+            McFly.dispatcher.waitFor([ProductStore.dispatcherID]);
+            _addToCart(payload.product);
+            CartStore.emitChange();
+            break;
+        case 'CART_CHECKOUT':
+            WebAPIUtils.checkoutProducts(payload.products);
```
Comment (voronianski) [inline mcfly/js/stores/CartStore.js:40]: @kenwheeler I prefer to not call APIs within stores.. is it possible to call `WebAPIUtils` in related action somewhere - https://github.com/voronianski/flux-comparison/pull/7/files#diff-0ae9121052ff6bc4e77cab865e4976feR18 ?

**Comment by kenwheeler** [comment]
Awesome, thanks man

**Comment by kenwheeler** [comment]
Hey I forgot to update the README. Do you want to , or do you want a PR?

**Comment by voronianski** [comment]
@kenwheeler I'll add, no problem.

**Comment by voronianski** [comment]
@kenwheeler done!

**Pull request merged**